### PR TITLE
add nooptracer

### PR DESCRIPTION
**Pull request by KushalD1999**

### CODE REVIEW

File: spotify_opentelemetry_wrapper.py [hunk 1]
```diff
@@ -176,6 +176,9 @@ def initialize_spotify_tracer(
     Initializes the Spotify auto instrumentors and tracer. This function should be called
     once at the start of your program.
     """
+    _disable_logging = os.getenv("OTEL_DISABLE_LOGGING", False)
```
Comment (aabramov-pc) [inline spotify_opentelemetry_wrapper.py:179]: This should be `disable_logging` - the underscore prefix is typically used to denote private class members

File: spotify_opentelemetry_wrapper.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from typing import Dict
 
 from opentelemetry import trace
-from opentelemetry.trace import NoOpTracerProvider
+from opentelemetry.trace import NoOpTracerProvider, NoOpTracer
```
Comment (aabramov-pc) [inline spotify_opentelemetry_wrapper.py:6]: This shouldn't be needed

File: spotify_opentelemetry_wrapper.py [hunk 3]
```diff
@@ -176,6 +176,9 @@ def initialize_spotify_tracer(
     Initializes the Spotify auto instrumentors and tracer. This function should be called
     once at the start of your program.
     """
+    _disable_logging = os.getenv("OTEL_DISABLE_LOGGING", False)
+    if _disable_logging:
+        return NoOpTracer()
```
Comment (aabramov-pc) [inline spotify_opentelemetry_wrapper.py:181]: This function doesn't return anything, you can just have `return` here to end execution. When callers try to use the `tracer` object, they will get the default-initialized NoOp tracer (see initialization above).

**Review by aabramov-pc — CHANGES_REQUESTED** [review]
We shouldn't return anything from this function.

File: spotify_opentelemetry_wrapper.py (hunk 3 above)
Comment (KushalD1999) [reply spotify_opentelemetry_wrapper.py]: got it, thanks

**Review by aabramov-pc — APPROVED** [review]

**Pull request merged**